src/scene/mesh.js

```diff
@@ -90,6 +90,8 @@ pc.extend(pc, function () {
      * </ul>
      * Defaults to pc.RENDERSTYLE_SOLID.
      * @property {Boolean} cull Controls whether the mesh instance can be culled with frustum culling
+     * @property {pc.StencilParameters} stencilFront Per-mesh instance stencil parameters for front faces (default null). Overrides any material settings.
+     * @property {pc.StencilParameters} stencilBack Per-mesh instance stencil parameters for back faces (default null). Overrides any material settings.

```

Comment:
Maybe don't make this public just yet. Get it deployed and see if it fixes what you need it for and we can always add it to public API later for everyone to enjoy. 😄